src/helpers/event/createEventHelpers/createRecurringEvent.ts

```diff
@@ -49,20 +48,14 @@ export const createRecurringEvent = async (
     data?.endDate
   );
 
-  const convertedStartDate = convertToUTCDate(data.startDate);
-  let convertedEndDate = null;
-  if (data.endDate) {
-    convertedEndDate = convertToUTCDate(data.endDate);
-  }
-
   // create a base recurring event first, based on which all the
   // recurring instances would be dynamically generated
   const baseRecurringEvent = await Event.create(
     [
       {
         ...data,
-        startDate: convertedStartDate,
-        endDate: convertedEndDate,
+        startDate: data.startDate,
+        endDate: data.endDate,
```

Comment:
are you assuming that sender would send the date in UTC format ? or should we perform the convertToUTCDate() here?

Reply:
Yes, the dates are formatted by already by dayjs, we're recieving dates in UTC. That's why I removed the unnecessary date formatting. I've been testing this for past couple of days.